ocean_lib/ocean/test/test_ocean_compute.py

```diff
@@ -0,0 +1,537 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import json
+from datetime import datetime
+
+from ocean_lib.assets.utils import create_checksum
+from ocean_lib.config import Config
+from ocean_lib.data_provider.data_service_provider import DataServiceProvider
+from ocean_lib.ocean.ocean import Ocean
+from ocean_lib.ocean.ocean_compute import OceanCompute
+from tests.resources.ddo_helpers import (
+    get_registered_algorithm_ddo,
+    wait_for_ddo,
+    get_registered_ddo_with_compute_service,
+)
+from tests.resources.helper_functions import get_publisher_wallet
+
+
+def test_build_cluster_attributes():
+    data_provider = DataServiceProvider
```

Comment:
please use the class directly, do not assign the class to a variable, because it looks like an instance. Same for below :)